src/components/attendee/VideoComms/internal/TwilioTrackDisplay.tsx

```diff
@@ -25,8 +25,9 @@ export const TwilioTrackDisplay: React.FC<TwilioTrackDisplayProps> = ({
   const videoRef = useRef<HTMLVideoElement>(null);
   useEffect(() => {
     if (videoRef.current) {
+      const elForCleanup = videoRef.current;
       track.attach(videoRef.current);
-      return () => void track.detach();
+      return () => void track.detach(elForCleanup);
     }
```

Comment:
The call to `detach` without any arguments will detach the Twilio video stream from **all** video elements. That's not what we want at all.

Reply:
@colinhowe Please add this comment inside the code prefixed with something like `// NOTE: `

Reply:
Resolved in 40c9750b0